src/System.Security.Permissions/src/System/Security/Permissions/MediaPermission.cs

```diff
@@ -0,0 +1,60 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+namespace System.Security.Permissions
+{
+    public enum MediaPermissionAudio
+    {
+        NoAudio,
+        SiteOfOriginAudio,
+        SafeAudio,
+        AllAudio
+    }
+    public enum MediaPermissionVideo
+    {
+        NoVideo,
+        SiteOfOriginVideo,
+        SafeVideo,
+        AllVideo,
+    }
+    public enum MediaPermissionImage
+    {
+        NoImage,
+        SiteOfOriginImage,
+        SafeImage,
+        AllImage,
+    }
+    [Serializable()]
+    [System.Runtime.CompilerServices.TypeForwardedFrom("WindowsBase, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35")]
```

Comment:
I'm not clear on when we have the TypeForwardedFrom attribute and when we don't.  There are only 4 of these in this library.

Reply:
`TypeForwardedFrom` is useful when a type is also `Serializable`. Imagine an object being serialized after it has been forwarded from `AssemblyOld` to `AssemblyNew`. We wouldn't want it to be serialized as type=`TypeName,Assembly=AssemblyNew`; we'd instead prefer that the object be serialzied as `Typename,Assembly=AssemblyOld`.

Reply:
Per @ericstj's advice, we are going to remove  `Serializable` (as well as `TypeForwardedFrom`), so this becomes moot...